tests/all.js

```diff
@@ -3,23 +3,24 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 define([
-  'tests/lib/init',
-  'tests/lib/signUp',
-  'tests/lib/signIn',
-  'tests/lib/verifyCode',
-  'tests/lib/request',
-  'tests/lib/hkdf',
-  'tests/lib/unbundle',
+  'tests/lib/account',
+  'tests/lib/certificateSign',
   'tests/lib/credentials',
+  'tests/lib/device',
+  'tests/lib/errors',
   'tests/lib/hawkCredentials',
+  'tests/lib/headerLang',
+  'tests/lib/hkdf',
+  'tests/lib/init',
+  'tests/lib/metricsContext',
+  'tests/lib/misc',
   'tests/lib/passwordChange',
-  'tests/lib/session',
-  'tests/lib/certificateSign',
-  'tests/lib/account',
   'tests/lib/recoveryEmail',
-  'tests/lib/misc',
-  'tests/lib/errors',
-  'tests/lib/headerLang',
+  'tests/lib/request',
+  'tests/lib/session',
+  'tests/lib/signIn',
+  'tests/lib/signUp',
+  'tests/lib/unbundle',
   'tests/lib/uriVersion',
-  'tests/lib/device'
+  'tests/lib/verifyCode'
```

Comment:
I took the liberty of re-ordering these as I felt like a short enough array to be reviewable and left things much easier to manage in the future. The added item is `'tests/lib/metricsContext'`, obviously.